src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        int randomIndex = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
        int randomIndex = random.nextInt(Color.values().length);
```
lets create a class level variable - to avoid creating a new instance each time this method is invoked